styles/components/_listingCard.scss

```diff
@@ -31,22 +36,10 @@
     align-items: center;
     justify-content: center;
     z-index: 1; // so it's above the overlay icons
-
-    .overlayPanelInner {
```

Comment:
Thinking about this, I may have borked this on the rebase potentially. So it might be my fault.